articles/partner-twilio-php-how-to-use-voice-sms.md

```diff
@@ -254,11 +278,16 @@ Now that you've learned the basics of the Twilio service, follow these links to
 [twiml]: http://www.twilio.com/docs/api/twiml
 [twilio_api]: http://www.twilio.com/api
 [try_twilio]: https://www.twilio.com/try-twilio
-[twilio_account]:  https://www.twilio.com/user/account
-[verify_phone]: https://www.twilio.com/user/account/phone-numbers/verified#
+[twilio_console]:  https://www.twilio.com/user/console
```

Comment:
these urls should be `https://www.twilio.com/console` instead of `https://www.twilio.com/user/console`.